ldap/servers/slapd/passwd_extop.c

```diff
@@ -287,7 +287,9 @@ passwd_modify_generate_policy_passwd(passwdPolicy *pwpolicy,
 
     /* if only minalphas is set, divide it into minuppers and minlowers. */
     if (pwpolicy->pw_minalphas > 0 &&
-        (my_policy[idx_minuppers] == 0 && my_policy[idx_minlowers] == 0)) {
+        (my_policy[idx_minuppers] == 0 && my_policy[idx_minlowers] == 0))
+    {
+        /* For Y2K38 we will need to handle time() returning a uint64_t value */
```

Comment:
I have the feeling that you missed my point: 
 For Y2K38 we do NOT have to do anything in this file:  time() will then return a uint64_t value that will then be truncated to a uint32_t but that is ok as we only want to generate 32 random  (more or less) bytes to initialize the pseudo random generator seed. (i.e: we do not care about the real date)
 The comment we should add is that we should ignore covscan Y2K38 warning. in theses two lines) !-;)
 
 Anyway I will approve the changes as it is as nothing is broken ...

Reply:
I added this above the two random function calls:

      /* coverity[store_truncates_time_t] */